scripts/components/Repo.js

```diff
@@ -1,19 +1,24 @@
 'use strict';
 
 import React, { PropTypes } from 'react';
-import PureRenderMixin from 'react/lib/ReactComponentWithPureRenderMixin';
+import shallowEqual from 'react/lib/shallowEqual';
 import { Link } from 'react-router';
 
-const Repo = React.createClass({
-  mixins: [PureRenderMixin],
+class Repo extends React.Component {
 
-  propTypes: {
-    repo: PropTypes.object.isRequired,
-    owner: PropTypes.object.isRequired
-  },
+  // TODO: make it either a HigherOrderComponent
```

Comment:
Or making it a subclass and extends from it? It would be a useful example for inheritance.